packages/react-components/tests/browser/RichTextSendBox.spec.tsx

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import React from 'react';
+import { expect } from '@playwright/experimental-ct-react';
+import { test as betaTest } from './FlavoredBaseTest';
+import { RichTextSendBox } from '../../src/components/RichTextEditor/RichTextSendBox';
+
+const formatButtonId = 'rich-text-input-box-format-button';
+
+betaTest.describe('RichTextSendBox tests', () => {
+  betaTest.skip(({ isBetaBuild }) => !isBetaBuild, 'The tests should be run for beta flavor only');
+  betaTest('RichTextSendBox should be shown correctly', async ({ mount }) => {
+    const component = await mount(
+      <RichTextSendBox
+        onSendMessage={async () => {
+          return;
+        }}
+      />
+    );
+    await component.evaluate(() => document.fonts.ready);
+    await expect(component).toHaveScreenshot('richTextSendBox-without-format-toolbar.png');
+    const formatButton = component.getByTestId(formatButtonId);
+
+    await formatButton.hover();
+    await expect(component).toHaveScreenshot('richTextSendBox-hover.png');
+
+    await formatButton.click();
+    //move mouse to the format button so the screenshots are consistent
+    await formatButton.hover();
+    await expect(component).toHaveScreenshot('richTextSendBox-with-format-toolbar.png');
+  });
+
+  betaTest('RichTextSendBox should be shown correctly when disabled', async ({ mount }) => {
+    const component = await mount(
+      <RichTextSendBox
+        onSendMessage={async () => {
+          return;
+        }}
+        disabled={true}
+      />
+    );
+    await component.evaluate(() => document.fonts.ready);
+    await expect(component).toHaveScreenshot('richTextSendBox-disabled-without-format-toolbar.png');
+  });
+
+  betaTest('RichTextSendBox should be shown system message correctly', async ({ mount }) => {
+    const component = await mount(
+      <RichTextSendBox
+        onSendMessage={async () => {
+          return;
+        }}
+        systemMessage={'Test system message'}
+      />
+    );
+    await component.evaluate(() => document.fonts.ready);
+    await expect(component).toHaveScreenshot('richTextSendBox-with-system-message.png');
+  });
+
+  betaTest('RichTextSendBox should show attachmentsWithProgress correctly', async ({ mount }) => {
+    const component = await mount(
+      <RichTextSendBox
+        onSendMessage={async () => {
+          return;
+        }}
+        attachmentsWithProgress={[
+          { progress: 0.65, id: 'id1', name: 'test1' },
+          { progress: 1, id: 'id2', name: 'test2' },
+          { progress: 0, id: 'id3', name: 'test3' }
```

Comment:
maybe add a extension here as well, so we can catch our layout issues

```
{ progress: 0.65, id: 'id1', name: 'test1.pdf', extension: 'pdf' },
{ progress: 1, id: 'id2', name: 'test2.docx', extension: 'docx' },
{ progress: 0, id: 'id3', name: 'test3'  }
```

also remember to init icons so they can show up:

```
import { initializeIcons } from "@fluentui/react";
import { initializeFileTypeIcons } from "@fluentui/react-file-type-icons";
initializeIcons();
initializeFileTypeIcons();
```

Reply:
Please note that icons for components tests should be initializad in `index.tsx` file in `packages/react-components/playwright/`